src/components/Checkout/SocialShareV2.vue

```diff
@@ -178,12 +198,7 @@ export default {
 					if (code !== '4201') {
 						this.$showTipMsg(`There was a problem sharing to Facebook: ${message}`, 'warning');
 					}
-					this.$kvTrackEvent(
```

Comment:
I'm pretty confident nobody has ever looked at this event. Simplifying it and making it the same across the files, but I'm not what useful data we can know from sending an event on this failure. If anything I think this should be a Sentry log event.